src/index.js

```diff
@@ -1,30 +1,21 @@
 import './style.css';
-import STORE from './Modules/store.js';
+import gamelist from './Modules/score.js';
+import games from './Modules/store.js';
 
-const { v4: uuidv4 } = require('uuid');
+const btn = document.querySelector('.btn');
+const name = document.querySelector('#name');
+const score = document.querySelector('#score');
+const submit = document.querySelector('#submit');
 
-const pieces = [
-  {
-    name: 'Joy',
-    score: '200',
-    id: uuidv4(),
-  },
+function list(i) {
```

Comment:
- [ ] Great job so far implementing some of the changes. like the previous reviewer noted, kindly use arrow functions instead of the `function` keyword as stated by the project requirement. you can see how to do that [here](https://www.w3schools.com/js/js_arrow_function.asp)